packages/primitives/navigation-menu/src/navigation-menu-indicator.directive.ts

```diff
@@ -0,0 +1,145 @@
+import { BooleanInput } from '@angular/cdk/coercion';
+import {
+    booleanAttribute,
+    computed,
+    Directive,
+    effect,
+    ElementRef,
+    inject,
+    input,
+    OnDestroy,
+    Renderer2,
+    runInInjectionContext,
+    signal,
+    untracked
+} from '@angular/core';
+import { injectNavigationMenu, isRootNavigationMenu } from './navigation-menu.token';
+
+@Directive({
+    selector: '[rdxNavigationMenuIndicator]',
+    host: {
+        '[attr.data-state]': 'isVisible() ? "visible" : "hidden"',
+        '[attr.data-orientation]': 'context.orientation',
+        '[style.display]': 'isVisible() ? null : "none"',
+        'aria-hidden': 'true'
+    }
+})
+export class RdxNavigationMenuIndicatorDirective implements OnDestroy {
+    private readonly context = injectNavigationMenu();
+    private readonly elementRef = inject(ElementRef);
+    private readonly renderer = inject(Renderer2);
+
+    /**
+     * Used to keep the indicator rendered and available in the DOM, even when hidden.
+     * Useful for animations.
+     * @default false
+     */
+    readonly forceMount = input<BooleanInput, unknown>(undefined, { transform: booleanAttribute });
```

Comment:
or `input<boolean, BooleanInput>` ?

Reply:
Weird... I think these were done automatically when I applied a legacy input --> signal migration. Perhaps I accidentally used the "unsafe" option. Nonetheless, fixed in d196f157989b27e5f10b859ec10c79af2a7d64b0.